applicationconfig/client/README.md

```diff
@@ -0,0 +1,84 @@
+# App Configuration client library for Java
+Azure App Configuration allows developers to store, retrieve and manage access to application settings all in one place. It is easy to set up and simple to use from any application. It gives developers the ability to modify an application's behavior on demand without having to redeploy the application.
+
+Use the client library for Application Configuration to:
+- Adding a new configuration
+- Updating an existing configuration
+- Retrieving a configuration
+- Deleting a configuration
+
+[Source code]() | [Package (Maven)]() | [API reference documentation]() | [Product documentation](https://docs.microsoft.com/en-us/azure/azure-app-configuration/)
+
+## Getting started
+
+### Prerequisites
+
+- JDK 8 or later
+- Maven
+- Azure Service Principal
```

Comment:
Not sure what you mean with this...
But take a look at: https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-aspnet-core-app#prerequisites

Something like that, plus whatever is needed for Java.

Reply:
still missing that an Azure subscription is needed. See the link I am providing where they list the prerequisites